src/hardware/driver/ltc2320.rs

```diff
@@ -0,0 +1,137 @@
+///! LTC2320 Driver
+///!
+///! QSPI bug (2.4.3):
+///! https://www.st.com/resource/en/errata_sheet/es0392-stm32h742xig-and-stm32h743xig-device-limitations-stmicroelectronics.pdf
+///!
+///! This driver is intended to be used in the following manner:
+///! 1. Trigger a new LTC2320 conversion with start_conversion(). This sets nCNV low and waits until TCONV has passed.
+///!    Then it pases the timer and starts the qspi readout.
+///! 2. Call handle_transfer_done_irq() in the QSPI ISR to retrieve the ADC data and set nCNV high again.
+///!
+///! Only works under the following condition:
+///! Conversions are not restarted faster than (T_readout + TCONV + TCNVH + readout/irq CPU overhead).
+///! You will always see an error
+use super::super::hal::{
+    device::QUADSPI,
+    gpio::{self, gpiob, gpioc, gpioe},
+    prelude::*,
+    rcc, stm32,
+    time::NanoSeconds,
+    timer::Timer,
+    xspi::{Qspi, QspiError, QspiMode, XspiExt},
+};
+use core::ptr;
+use fugit::Hertz;
+use nb::block;
+
+#[derive(Copy, Clone, Debug)]
+pub enum Error<E> {
+    TimerBusyError,
+    QspiError(E),
+}
+
+impl<E> From<E> for Error<E> {
+    fn from(err: E) -> Error<E> {
+        Error::QspiError(err)
+    }
+}
+
+pub struct Ltc2320Pins {
+    pub spi: (
+        gpiob::PB2<gpio::Alternate<9>>,
```

Comment:
Is that really a different AF compared to  the other pins?

Reply:
Jes, just double checked it in the datasheet.